docker-compose.yaml

```
@@ -9,42 +9,57 @@ services:
       POSTGRES_DB: airflow
     volumes:
       - postgres-db-volume:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U airflow"]
+      interval: 10s
+      retries: 5
 
   airflow-webserver:
+    build: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)